Update available models, multiple bug fixes

kaklik opened the pull request:
These commits probably resolve the following issues: 

- https://github.com/ThunderFly-aerospace/PX4-FlightGear-Bridge/issues/22
- https://github.com/ThunderFly-aerospace/PX4-FlightGear-Bridge/issues/10
- Enables RPM message from autogyro to PX4.
- Adds [TF-G2 autogyro](https://github.com/ThunderFly-aerospace/FlightGear-TF-G2) model.

roman-dvorak commented:
I did a rebase and straightened the branch. 

It has been tested to be compatible with https://github.com/PX4/PX4-Autopilot/pull/19122 PR] and the TF-G2 autogyro flightgear model works here. 

It looks like it's ready to merge and the simulation will work in the master version of the autopilot FW.

roman-dvorak commented:
Hi @Jaeyoung-Lim, can you please do a review?

Jaeyoung-Lim commented:
@roman-dvorak The git history seems to be broken with a possibly in correct rebase of the `ThunderFly-aerospace/master` branch. Is there any chance you did a rebase and force push on the upstream(`ThunderFly-aerospace/PX4-FlightGear-Bridge`) repo?

For example, https://github.com/PX4/PX4-FlightGear-Bridge/pull/6/commits/553fa9861997077d665adb3df9fb6cbbde9a0cf0 is part of https://github.com/PX4/PX4-FlightGear-Bridge/pull/1, but this PR is trying to add a redundant commit with different contents

roman-dvorak commented:
Thanks Lim, I did a rebase so that the branch was straight (Because it consisted of many branches) I will try to correct these shortcomings. But I don't quite understand how this is possible.

roman-dvorak committed:
Add support of TF-G1 and TF-G2 to FG bridge, update readme, update mavlink communication
Add support of TF-G1 and TF-G2 to FG bridge, update readme, update mavlink communication

Jaeyoung-Lim commented:
@roman-dvorak Yes exactly, force pushing to master branches can cause these problems. 

What were the intentions of rebasing on top of this repo instead of moving forward with ThunderflyAerospace Flight Gear? Was the intention trying to make this the upstream repo?

roman-dvorak commented:
@Jaeyoung-Lim, I went through the whole structure and I did squash there. This solved the problem with duplicate commits that got there by various cross-mergers between multiple branches.

Is it now ready to merge?  

This was due to various testing and development. Because we needed to test our modifications (in models submodules) with the latest master firmware.

Jaeyoung-Lim reviewed (approved):
Thanks!

LGTM
